src/sentry/static/sentry/app/views/onboarding/welcome.tsx

```diff
@@ -1,77 +1,130 @@
+  t("We weren't lying to you, time to onboard"),
+];
+
+const fadeAway: MotionProps = {
+  variants: {
+    initial: {opacity: 0},
+    animate: {opacity: 1, filter: 'blur(0px)'},
+    exit: {opacity: 0, filter: 'blur(1px)'},
+  },
+  transition: {duration: 0.8},
+};
+
+const OnboardingWelcome = ({organization, onComplete, active}: Props) => {
   const skipOnboarding = () => recordAnalyticsOnboardingSkipped({organization});
 
   return (
-    <React.Fragment>
-      <p>
-        {tct("We're happy you're here, [name]!", {
-          name: <strong>{user.name.split(' ')[0]}</strong>,
-        })}
-      </p>
-      <p>
-        {t(
-          `With Sentry, find and fix bugs and hunt down performance slowdowns
-             before customers even notice a problem. When things go to hell
-             we’ll help fight the fires. In the next two steps you will…`
-        )}
-      </p>
-      <ul>
-        <li>{t('Choose your platform.')}</li>
-        <li>
-          {t(
-            `Integrate Sentry into your application, invite your team, or take
-               a tour of Sentry.`
-          )}
-        </li>
-      </ul>
-      <ActionGroup>
-        <Button
-          data-test-id="welcome-next"
-          disabled={!active}
-          priority="primary"
-          onClick={() => onComplete({})}
-        >
-          {t("I'm Ready!")}
-        </Button>
-        <SecondaryAction>
-          {tct('Not your first Sentry rodeo? [exitLink:Skip this onboarding].', {
-            exitLink: <Button priority="link" onClick={skipOnboarding} href="/" />,
-          })}
-        </SecondaryAction>
-      </ActionGroup>
-    </React.Fragment>
+    <FallingError
+      onFall={fallCount => fallCount >= easterEggText.length && onComplete({})}
+    >
+      {({fallingError, fallCount, triggerFall}) => (
+        <Wrapper>
+          <WelcomeBackground />
+          <motion.h1 {...fadeAway}>{t('Welcome to Sentry')}</motion.h1>
+          <motion.p {...fadeAway}>
+            {t(
+              'Find the errors and performance slowdowns that keep you up at night. In two steps.'
+            )}
+          </motion.p>
+          <CTAContainer {...fadeAway}>
+            <Button
+              data-test-id="welcome-next"
+              disabled={!active}
+              priority="primary"
+              onClick={() => {
+                triggerFall();
+                onComplete({});
+              }}
+            >
+              {t("I'm Ready")}
+            </Button>
+            <PositionedFallingError>{fallingError}</PositionedFallingError>
+          </CTAContainer>
+          <SecondaryAction {...fadeAway}>
+            {tct('[flavorText][br][exitLink:Skip onboarding].', {
+              br: <br />,
+              exitLink: <Button priority="link" onClick={skipOnboarding} href="/" />,
+              flavorText:
+                fallCount > 0
+                  ? easterEggText[fallCount - 1]
+                  : t('Geez Mom, I’ve used Sentry before.'),
```

Comment:
Should this be an ascii quote?
```suggestion
                  : t("Geez Mom, I've used Sentry before."),
```